src/GraphQL/GraphQL.csproj

```diff
@@ -24,8 +24,11 @@
   </ItemGroup>
 
   <ItemGroup>
+    <Reference Include="Microsoft.CSharp"/>
+    <Reference Include="System" />
+    <Reference Include="System.Core" />
```

Comment:
This project multi-targets several frameworks.  These need to stay under the `net45` umbrella.

Reply:
Afaik netore handles this scenario correctly. Ie it will redirect to the correct assemblies at run time 

Also when i left it the 45 condition it would not compile

Reply:
Is that a new feature with the new SDK?  It appears CI is failing specifically because of that change.  It is using `2.1.4` SDK.

https://github.com/graphql-dotnet/graphql-dotnet/blob/master/global.json

Reply:
Yep. I updated the sdk in my netcore 2.1 pr. I will rebase this one after netcore is merged